module/exporter-shopware-6/src/Infrastructure/Synchronizer/CategorySynchronizer.php

```diff
@@ -222,7 +217,9 @@ private function getShopwareCategory(Shopware6Channel $channel, string $name): ?
      */
     private function getShopwareCategoryList(Shopware6Channel $channel): array
     {
-        $action = new GetCategoryList();
+        $query = new SwagQLBuilder();
+        $query->limit(500);
```

Comment:
What happen if there is more then 500 categories ??

Reply:
Because, default API value is 10 or 20